GuessingGame/Service.cs

```diff
@@ -11,9 +11,9 @@ public Service(INumberGenerator numberGenerator)
         {
             _numberGenerator = numberGenerator;
         }
-        
 
-        public int GetAmount()
+
+    public int GetAmount()
     {
     Console.WriteLine("Enter the amount of money to bet:");
```

Comment:
If this was production code you would usually indent this part of the code it would make it easier to read.